src/modules/uORB/ORBMap.hpp

```diff
@@ -90,7 +96,7 @@ class uORB::ORBMap
 		}
 
 		_end->next = nullptr;
-		_end->node_name = strdup(node_name);
```

Comment:
I'd say it's save, since:
- the API doc clearly states the condition
- ORBMap is only used within uORB, for a single set of objects, with a lifetime over the whole runtime, ie never deleted (would we want to start deleting them we would have to do some deeper changes anyway to guarantee thread-safety and correctness)